CHANGE-NOTES.html

```diff
@@ -1 +1,3 @@
+<p>Add support for IntelliJ 2022.2.</p>
+
 <p>Show a path to repository (from repository root) instead of a repository name alone in the repository selection combobox.</p>
```

Comment:
Btw @mkondratek isn't is supposed to be:
```suggestion
<p>Show a path to repository (from project root) instead of a repository name alone in the repository selection combobox.</p>
```
❓

Reply:
Hmmm well... but what if an IntelliJ project contains a couple different git repositories? I mean... a path _from repository root_ to the given repository is technically always empty (`.`), but the path from IntelliJ project root will contain 1+ fragment

Reply:
I suppose you meant _project_ here... lmk if needs to be reverted